doc/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ if(${LATEXMK} STREQUAL LATEXMK-NOTFOUND)
 endif()
 
 if(NOT DEFINED SPHINXOPTS)
-  set(SPHINXOPTS -q -j auto)
+  set(SPHINXOPTS -j auto)
```

Comment:
This is making the output for the documentation build step in CI pretty useless; can you tweak this so that is done the same way, with -q etc.?

Reply:
I'll add `-q` to CI